main/getting-started/Documentation-Map.md

```diff
@@ -0,0 +1,69 @@
+# Agoric Documentation Map
+
+**tyg todo**: Add links to all these docs before publishing
+
+Agoric provides three main types of documentation, as well as links to
+"big picture" presentations and papers and links to our source code on GitHub. This page
+describes our documentation structure, where documents are, and when you should use them, 
+as well as what order to read them.
+
+Our three main documentation types are:
+- **Getting Started**: These are the first things you should
+read. They cover what to do to install, set up, and start
+using Agoric's smart contract tools. They also introduce those tools and
+their underlying concepts and designs.
+- **Guides**: These are more detailed and complete descriptions
+of our tools and their concepts, designs, and usage. They include
+descriptions and examples of how to do various tasks with Agoric
+software.
+- **Reference**: These are the specs for our APIs and commands,
+  as well as checklist tables that specify and explain the
+  steps for extended procedures, such as installing Agoric
```

Comment:
Should we imply that installing Agoric software is an extended procedure? Maybe: "and explain the procedure for tasks such as installing and using the Agoric software."